jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/JavascriptCodeWhispererFileCrawler.kt

```diff
@@ -9,15 +9,16 @@ import com.intellij.psi.PsiFile
 object JavascriptCodeWhispererFileCrawler : CodeWhispererFileCrawler() {
     override val fileExtension: String = "js"
     override val dialects: Set<String> = setOf("js", "jsx")
-
-    override val testFilenamePattern: Regex = """^.*\.test\.(js|jsx)${'$'}""".toRegex()
-
-    // TODO: Add implementation when UTG is enabled
-    override fun guessSourceFileName(tstFileName: String): String = ""
```

Comment:
moved to parent class https://github.com/aws/aws-toolkit-jetbrains/pull/3774/files#diff-faff4822410665507967aaa15bfe64abab03286e73a1bdc9c9d5046408de467bR139